tests/lib/rules/use-isnan.js

```diff
@@ -262,18 +262,24 @@ ruleTester.run("use-isnan", rule, {
             code: "foo.indexOf(a, NaN)",
             options: [{ enforceForIndexOf: true }]
         },
-        {
-            code: "foo.lastIndexOf(NaN, b)",
-            options: [{ enforceForIndexOf: true }]
-        },
+
+        /*
+         * {
+         *     code: "foo.lastIndexOf(NaN, b)",
+         *     options: [{ enforceForIndexOf: true }]
+         * },
+         */
```

Comment:
Can you uncomment these tests and add 3rd argument that would make them valid?

Reply:
Done!

Reply:
@mdjermanovic, in this rule following code
```js
myArray.indexOf(NaN)
```
fixes to 
```js
myArray.findIndex(Number.isNaN)
```

but both cases return `-1` so is this fix correct or there is something i am not aware of?

Reply:
The fix is correct. For example:

```js
const myArray = [0, 1, 2, NaN, 4, 5];

console.log(myArray.indexOf(NaN)); // -1

console.log(myArray.findIndex(Number.isNaN)); // 3

```

Reply:
Got it, thanks!